src/azul/__init__.py

```diff
@@ -244,7 +244,7 @@ def api_lambda_domain_aliases(self, lambda_name):
         """
         Additional alias domain names for the given API lambda
         """
-        return [self.drs_domain] if lambda_name == 'service' and self.drs_domain else []
+        return [self.drs_endpoint()] if lambda_name == 'service' and self.drs_domain else []
```

Comment:
An endpoint is a URL. A domain is just a hostname. This method needs to return domains, not URLs. If you look at the callsite, it's in the TF config. Did you test this change by running `make terraform`?

Reply:
That would have revealed the mistake.

Reply:
I did not test it that way. I had conflated, "endpoint" and "domain". I see my mistake and removed this commit.